scripts/exgdas_enkf_sfc.sh

```diff
@@ -155,10 +158,17 @@ if [ $DOIAU = "YES" ]; then
             ${NLN} "${FIXorog}/${CASE}/${CASE}_grid.tile${n}.nc"     "${DATA}/fngrid.${cmem}"
             ${NLN} "${FIXorog}/${CASE}/${CASE}.mx${OCNRES}_oro_data.tile${n}.nc" "${DATA}/fnorog.${cmem}"
 
-        done
-
-        CDATE="${PDY}${cyc}" ${CYCLESH}
-        export err=$?; err_chk
+            if [[ ${GSI_SOILANAL} = "YES" ]]; then
+                FHR=6
+                ${NLN} "${COM_ATMOS_ANALYSIS_MEM}/${APREFIX_ENS}sfci00${FHR}.nc" \
```

Comment:
```suggestion
                ${NLN} "${COM_ATMOS_ANALYSIS_MEM}/${APREFIX_ENS}sfci006.nc" \
```

Reply:
the consistent change would be this becomes `sfcinc.nc`, I suggest if we want `sfci006` that we do that in a subsequent PR and do it system-wide.